app/src/org/commcare/dalvik/activities/CommCareHomeActivity.java

```diff
@@ -201,8 +203,8 @@ protected void returnToLogin() {
     }
 
     protected void attemptSync() {
-        if (isNetworkNotConnected()) {
-            if (isAirplaneModeOn()) {
+        if (ConnectivityStatus.isNetworkNotConnected(CommCareHomeActivity.this)) {
```

Comment:
CommCareActivity already has an isNetworkNotConnected() method, which calls out to a method in AndroidUtil. Would be nice to use that here, and then maybe move the isAirplaneModeOn() method into AndroidUtil and call it in a similar way, so that we don't need the extra util class. We definitely already have both of these functionalities scattered across a few different places in the code right now. Would be good to try to keep it unified going forward, and eventually do a real pass at eliminating duplicates.